src/agent/samplers/cpu/linux/bandwidth/stats.rs

```diff
@@ -29,3 +29,24 @@ pub static CGROUP_CPU_THROTTLED_TIME: CounterGroup = CounterGroup::new(MAX_CGROU
     metadata = { unit = "events" }
 )]
 pub static CGROUP_CPU_THROTTLED: CounterGroup = CounterGroup::new(MAX_CGROUPS);
+
+#[metric(
+    name = "cgroup_cpu_bandwidth",
```

Comment:
Metrics with the same name should be measuring the same thing where some key-value metadata is used to distinguish between types of that same thing. For instance: `blockio_operation` can be categorized as `read` or `write` but summing them together gives us total operations (which is also a useful way to look at that metric).

Reply:
fixed